Automattic/simplenote-electron
# Fixes shortcut to enable Markdown preview

### Fix
Fixes #1387 Ctrl/Cmd + Shift + p is supposed to enable the markdown preview. There were two
errors. We were checking for a capital 'P' when we should have been checking
for 'shift' and lowercase 'p'. We were also using a bad check for markdown.

### Test
1. Open prod to note with markdown enabled
2. Cmd + Shift + P
3. Observe nothing happens
4. Do the same on this branch
5. Observe markdown preview toggles

### Review
Only one developer is required to review these changes, but anyone can perform the review.

### Release
`RELEASE-NOTES.txt` was updated with:

- Fixed keyboard shortcut to toggle markdown preview
